app/src/test/java/com/example/workoutapp/WorkoutUnitTest.kt

```diff
@@ -58,49 +53,34 @@ class WorkoutUnitTest {
     //This test makes sure the dto is gathering the info from the json service we are using
     @Test
     fun `Given a dto when name is Barbell Curl, and level is beginner`() {
-        var workout = Workout("Barbell Curl", "pull", "beginner", "isolation", "barbell", "biceps", "forearms",
-            "Stand up with your torso upright while holding a barbell at a shoulder-width grip. The palm of your hands should be facing forward and the elbows should be close to the torso. This will be your starting position.",
-            "While holding the upper arms stationary, curl the weights forward while contracting the biceps as you breathe out. Tip: Only the forearms should move.",
-            "Continue the movement until your biceps are fully contracted and the bar is at shoulder level. Hold the contracted position for a second and squeeze the biceps hard.",
-            "Slowly begin to bring the bar back to starting position as your breathe in.",
-            "Repeat for the recommended amount of repetitions.", "strength")
+        val workout = Workout("Barbell Curl", "pull", "beginner", "isolation", "barbell", "biceps", "forearms",
```

Comment:
I'd move this to strings.xml, or some other configuration file (like a JSON feed that you read.)   A) Don't assume everyone speaks English.  B) Hardcoded text means the entire application has to be recompiled, and a new version released, for a simple text change.